www/js/app.js

```diff
@@ -634,36 +634,36 @@ define(['jquery', 'zimArchiveLoader', 'uiUtil', 'settingsStore','abstractFilesys
         var messageChannelStatus = document.getElementById('messageChannelStatus');
         var serviceWorkerStatus = document.getElementById('serviceWorkerStatus');
         if (isMessageChannelAvailable()) {
-            messageChannelStatus.innerHTML = 'MessageChannel API available';
+            messageChannelStatus.textContent = 'MessageChannel API available';
             messageChannelStatus.classList.remove('apiAvailable', 'apiUnavailable');
             messageChannelStatus.classList.add('apiAvailable');
         } else {
             apiPanelClass = 'card-warning';
-            messageChannelStatus.innerHTML = 'MessageChannel API unavailable';
+            messageChannelStatus.textContent = "MessageChannel API unavailable";
```

Comment:
Why did you change the quotation style? This is explicitly mentioned in CONTRIBUTING.md... You should keep the quotation style of the area you are editing, for consistency.